broker/ingestexportservice.py

```diff
@@ -113,31 +118,31 @@ def secondarySubmission(self, submissionEnvelopeUuid, analyses):
                                                            "submittedName": file_json["fileName"],
                                                            "url": file_json["cloudUrl"],
                                                            "dss_uuid": file_json["uuid"]["uuid"],
-                                                           "indexed" : False,
-                                                           "content-type" : '"metadata/data"'
+                                                           "indexed" : False
                                                            }, files))
 
             # stage the analysis.json, add to filesToTransfer and to the bundle manifest
             analysisUuid = analysis["uuid"]["uuid"]
             analysisDssUuid = unicode(uuid.uuid4())
             analysisBundleContent = self.getBundleDocument(analysis)
             analysisFileName = "analysis_0.json" # TODO: shouldn't be hardcoded
-            fileDescription = self.writeMetadataToStaging(submissionEnvelopeUuid, analysisFileName, analysisBundleContent, "\"metadata/analysis\"")
 
             bundleManifest.fileAnalysisMap = { analysisDssUuid : [analysisUuid] }
-            filesToTransfer.append({"name":analysisFileName,
+
+            if not self.dryrun:
+                fileDescription = self.writeMetadataToStaging(submissionEnvelopeUuid, analysisFileName, analysisBundleContent, "hca-analysis")
```

Comment:
The last parameter here should be "\"metadata/analysis\""

Reply:
Fixed.

Reply:
Ok, let's not worry about replacing flatten.py for now but flag it as something to look at next time.

Reply:
Ok, let's not worry about replacing flatten.py for now but flag it as something to look at next time.